playbook-website/app/views/guides/getting_started/swift_setup.md

```diff
@@ -30,24 +29,15 @@ Built and maintained by the User Experience Team at [Power Home Remodeling](http
     ```
 5. Build the project
 
-### Install [`pre-commit`](https://pre-commit.com/#install) via `brew install pre-commit`
-  1. In the Playbook Swift root directory, run `pre-commit install` to set up the git hook scripts
+# Code Format
 
-# Code Format & Linting
-
-See the [Editing Linting, Formatting, and Auto Correction](LINTING.md) doc for more information.
+Playbook Swift enforces an indentation width of 2 spaces.
 
 ## Component Examples via PlaybookShowcase
 
 Playbook Swift contains the app named `PlaybookShowcase`; providing examples of each component. PlaybookShowcase can be launched within the XCode simulator, side-loaded, or downloaded to your device.
```

Comment:
The Playbook Swift repo contains the `PlaybookShowcase` app, which provides interactive examples of each component.